modules/api/src/main/scala/cards/nine/app/NineCardsApiActor.scala

```diff
@@ -15,32 +15,36 @@
  */
 package cards.nine.app
 
-import akka.actor.Actor
+import akka.actor.ActorRefFactory
+import akka.event.LoggingAdapter
 import cards.nine.api.{ AuthHeadersRejectionHandler, NineCardsExceptionHandler, NineCardsRoutes }
 import cards.nine.commons.config.NineCardsConfig
-import spray.routing.HttpService
-import spray.routing.authentication.{ BasicAuth, BasicHttpAuthenticator, UserPass }
 
 import scala.concurrent.ExecutionContext
 
-class NineCardsApiActor
-  extends Actor
-  with AuthHeadersRejectionHandler
-  with HttpService
+// TODO rename or merge into `NineCardsRoutes`
+class NineCardsApiActor(
```

Comment:
Now this class isn't an actor and its only job is to provide the config and rejection/exception handlers.